devel/python/python/ert/enkf/enkf_fs_manager.py

```diff
@@ -236,8 +243,13 @@ def initializeCaseFromScratch(self, case, parameter_list, from_iens, to_iens, fo
         @type from_iens: int
         @type force_init: bool
         """
-        self._initialize_from_scratch(case, parameter_list, from_iens, to_iens, force_init)
+        mask = BoolVector( initial_size = self.getEnsembleSize(  ) , default_value = False )
+        for iens in range(from_iens,to_iens+1):
+            mask[iens] = True
+            
+        self._initialize_from_scratch(case, parameter_list, mask , force_init)
```

Comment:
This may be correct but it seems strange: `force_init = True = 1 = INIT_CONDITIONAL` 
`INIT_FORCE = 2`